### PR TITLE
Add Max-Forwards Header

### PR DESCRIPTION
**Pull request by bhaveshraj336**
#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/http/headers/max-forward/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+tags:
+  - HTTP
+  - HTTP Header
+  - Reference
+  - Request header
+browser-compat: http.headers.Accept
+---
+{{HTTPSidebar}}
+
+The **`Max-Forwards`** request HTTP header is used only with the TRACE method,to limit the number of proxies that request goes through.A Integer Value is given to these Header.If a TRACE Request is received with the Max-Forwards method,each hop should decrement it and then forward the request to next hop.If the received value is zero (and so cannot be deincremented further), the node becomes authortative for the given request, instead of the origin server.If the Max-Forwards Header is missing in a TRACE Request ,assume that there is no maximum number of forwards.
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:13]:
```suggestion
The **`Max-Forwards`** request HTTP header is used with the [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method to limit the number of nodes (usually proxies) that request goes through. Its value is an integer value indicating the _maximum amount_ of nodes it must visit. At each node, the value is decremented and the `TRACE` request is forwarded to the next node, until the destination is reached, or the received value of `Max-Forwards` is zero. The request is then forwarded back, except for some headers, as the body of a `200 OK` response.

If the `Max-Forwards` header is not present in a `TRACE` request, a node will assume that there is no maximum number of forwards.
```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <th scope="row">{{Glossary("Forbidden header name")}}</th>
+      <td>no</td>
+    </tr>
+  </tbody>
+</table>
+
+## Syntax
+
+```http
+Max-Forwards: <Integer-Value>
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:31]:
```suggestion
Max-Forwards: <integer>

```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+```
+## Examples
+
+```http
+Max-Forwards: 0
+
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:37]:
```suggestion
```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Max-Forwards: 10
+
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:39]:
```suggestion
```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- HTTP [content negotiation](/en-US/docs/Web/HTTP/Content_negotiation)
+- [List of default Accept values](/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values)
+- A header with the result of the content negotiation: {{HTTPHeader("Content-Type")}}
+- Other similar headers: {{HTTPHeader("TE")}}, {{HTTPHeader("Accept-Encoding")}}, {{HTTPHeader("Accept-Language")}}
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:55]:
```suggestion
- The HTTP [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/http/headers/max-forward/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+tags:
+  - HTTP
+  - HTTP Header
+  - Reference
+  - Request header
+browser-compat: http.headers.Accept
+---
+{{HTTPSidebar}}
+
+The **`Max-Forwards`** request HTTP header is used with the [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method to limit the number of nodes (usually proxies) that request goes through. Its value is an integer value indicating the _maximum amount_ of nodes it must visit. At each node, the value is decremented and the `TRACE` request is forwarded to the next node, until the destination is reached, or the received value of `Max-Forwards` is zero. The request is then forwarded back, except for some headers, as the body of a `200 OK` response.
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:13]:
```suggestion
The **`Max-Forwards`** request HTTP header is used with the [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method to limit the number of nodes (usually proxies) that request goes through. Its value is an integer value indicating the _maximum amount_ of nodes it must visit. At each node, the value is decremented and the `TRACE` request is forwarded to the next node, until the destination is reached, or the received value of `Max-Forwards` is zero. The request is then sent back, except for some headers, as the body of a `200 OK` response.

```

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/web/http/headers/max-forward/index.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+title: Max-Forwards
+slug: Web/HTTP/Headers/Accept
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:3]:
```suggestion
slug: Web/HTTP/Headers/Max-Forwards
```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
+---
+title: Max-Forwards
+slug: Web/HTTP/Headers/Accept
+tags:
+  - HTTP
+  - HTTP Header
+  - Reference
+  - Request header
+browser-compat: http.headers.Accept
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:9]:
```suggestion
browser-compat: http.headers.Max-Forwards
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
The name of the file is wrong, it is missing a _s_: it is _max-forwards_.

File: files/en-us/web/http/headers/max-forward/index.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  - HTTP Header
+  - Reference
+  - Request header
+browser-compat: http.headers.Max-Forwards
+---
+{{HTTPSidebar}}
+
+The **`Max-Forwards`** request HTTP header is used with the [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method to limit the number of nodes (usually proxies) that request goes through. Its value is an integer value indicating the _maximum amount_ of nodes it must visit. At each node, the value is decremented and the `TRACE` request is forwarded to the next node, until the destination is reached, or the received value of `Max-Forwards` is zero. The request is then sent back, except for some headers, as the body of a `200 OK` response.
+
+
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:15]:
```suggestion

```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+      <td>no</td>
+    </tr>
+  </tbody>
+</table>
+
+## Syntax
+
+```http
+Max-Forwards: <integer>
+
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:34]:
```suggestion
```

File: files/en-us/web/http/headers/max-forward/index.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- The HTTP [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forward/index.md:52]:
```suggestion
- The HTTP [`TRACE`](/en-US/docs/Web/HTTP/Methods/TRACE) method

```

**Comment by bhaveshraj336** [comment]
@teoli2003 can you plz let me know how can I change the name without creating a new pull request?

**Comment by teoli2003** [comment]
It is easy. See: https://docs.github.com/en/repositories/working-with-files/managing-files/moving-a-file-to-a-new-location

**Comment by bhaveshraj336** [comment]
@teoli2003 Thanks 
I have done the changes ?

**Comment by bhaveshraj336** [comment]
@teoli2003 so could you merge now?

File: files/en-us/web/http/headers/max-forwards/index.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+      <td>no</td>
+    </tr>
+  </tbody>
+</table>
+
+## Syntax
+
+```http
+Max-Forwards: <integer>
+```
```
Comment (teoli2003) [inline files/en-us/web/http/headers/max-forwards/index.md:34]:
```suggestion
```

```

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by bhaveshraj336** [comment]
@teoli2003  BTW why its not updated on the site?

**Comment by bhaveshraj336** [comment]
@teoli2003  Thank you for helping me out .

**Comment by bhaveshraj336** [comment]
@teoli2003  will i also need to update the main headers page?

**Comment by teoli2003** [comment]
Yes, please, this would be helpful!

**Comment by teoli2003** [comment]
> @teoli2003 BTW why its not updated on the site?

The site is regenerated about once a day, so it will be easier today (we are Sunday, I don't know if it will), or tomorrow night.

**Comment by bhaveshraj336** [comment]
> > @teoli2003 BTW why its not updated on the site?
> 
> The site is regenerated about once a day, so it will be easier today (we are Sunday, I don't know if it will), or tomorrow night.
and what's the BCD flaw?

**Comment by teoli2003** [comment]
> > > @teoli2003 BTW why its not updated on the site?
> > 
> > 
> > The site is regenerated about once a day, so it will be easier today (we are Sunday, I don't know if it will), or tomorrow night.
> > and what's the BCD flaw?

We don't have the BCD information at mdn/browser-compat-data about Max-Forwards. I will probably create a follow-up PR to fix it, but I didn't want to hold this PR.

**Comment by bhaveshraj336** [comment]
Oh Thats Great
Thanks again 😄

**Pull request merged**